src/main/webapp/index.html

```diff
@@ -15,15 +15,18 @@
   <script src="state-info.js"></script>
   <script src="data-visualization.js"></script>
   <script src="query-control.js"></script>
+  <script src="login-status.js"></script>
   <script src="user-location.js"></script>
   <link rel="stylesheet" href="style.css">
   <link id="favicon" rel="shortcut icon" href="./images/favicon.ico">
 </head>
 
-<body onload="registerServiceWorker(); createStateDropdownList(); loadAppropriateIcon(false);">
+<body
+  onload="registerServiceWorker(); createStateDropdownList(); loadAppropriateIcon(false); getAndDisplayLoginStatus();">
   <main>
     <div id="header">
-      <h1><a href="/">Census Viz</a>
+      <h1><a href="/">Census Viz</a></h1>
+      <div id="header-buttons">
```

Comment:
The buttons used to all be vertically aligned with each other, and now they're not. I'm not sure exactly why but it definitely is off.

Reply:
Imo there's also a bit too much space above the header (more than is below the words/icons), so that it doesn't look as symmetrical, but I think that's more subjective